mattdeann/urban-native
# Baf/feature/crop details component

### What does this PR do?
- This PR creates a CropDetails component with a Route so a user can click on an individual "seed packet" and it will take them to another view where they can see all the details of a crop such as description, sun requirements, days to maturity etc.

### What does it fix?
- No bugs to fix at this point.

### Is this a feature or a fix?
- This is a feature, being able to click on a given crop and view all of the details for that crop.

### Where should the reviewer start?
- Clown down this repository as well as the urban-native-api repository. CD into each of these directories in your command line. Run nodemon server.js in the api directory, and run npm start in this directory. Make sure to install react-router-dom as a dependency.

### How should this be tested?
- Once the servers are running and the page is displayed, try clicking on a crop and see the crop details view pop up. The back button does not have functionality yet, so you can hit the back button to navigate back to the home page and try other crops as well.


#### Please review considerations below:
